pkg/kube/namespace/filter.go

```diff
@@ -74,6 +74,10 @@ func (d *discoveryNamespacesFilter) Filter(obj any) bool {
 		return true
 	}
 
+	if ns, ok := obj.(string); ok {
```

Comment:
This is to filter out namespaces, which is useful for namespace controller

Reply:
Its not used in this PR, right? Just the future?

In the namespace controllre PR we discussed making a `FilterObject(obj Object)` and `FilterNamespace(namespace string)`. Instead of doing all the type casting, etc. This is more explicit API

Reply:
removed